### PR TITLE
chore: text changes

### PR DESCRIPTION
**Pull request by BCerki**
https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=49836037

There's a bug where the point of contact (previously called application lead) checkbox is backwards: https://github.com/bcgov/cas-registration/issues/649

Suggested review process:
- get on a screenshare with Nan and check all the text changes against the wireframes (Note: some text changes are handled in separate tickets (listed in the issue^). If a change seems to be missing in this PR, it might be covered by another ticket.)
- I manually tested submitting the operator and operator forms to make sure the renaming didn't break anything, but feel free to double check

### CODE REVIEW

File: client/app/components/routes/select-operator/form/UserOperator.tsx [hunk 1]
```diff
@@ -104,11 +105,22 @@ export default async function UserOperator({
...
-      formData={formData}
-    />
+    <>
+      <Note
+        classNames={"mb-4 mt-6"}
+        showNotePrefix={false}
+        showAlertIcon={true}
+        message="Please fill out missing information or update incorrect information about your operator to continue with your application.
+Some fields cannot be edited. If you need to change those fields, please contact us via email at GHGRegulator@gov.bc.ca."
+      />
```
Comment (Sepehr-Sobhani) [inline client/app/components/routes/select-operator/form/UserOperator.tsx:119]: Should we display this component only when the user needs to update an existing operator?

File: client/app/components/routes/select-operator/form/UserOperator.tsx [hunk 2]
```diff
@@ -104,11 +105,22 @@ export default async function UserOperator({
 
   return params?.id === "request-access" ? (
     // If the operator exists then show the form from the second page
-    <UserOperatorContactForm formData={formData} schema={userOperatorPage2} />
+    <>
```
Comment (Sepehr-Sobhani) [inline client/app/components/routes/select-operator/form/UserOperator.tsx:108]: Is this Fragment required?

**Review by Sepehr-Sobhani — COMMENTED** [review]
Awesome! I'm posting my comments here until I reach out to Nan for the screen-sharing session.

File: bc_obps/registration/api/operation.py [hunk 3]
```diff
@@ -234,23 +234,23 @@ def update_operation(request, operation_id: int, submit: str, payload: Operation
             },
         )
         al.set_create_or_update(modifier=user)
-        operation.application_lead = al
+        operation.point_of_contact = al
```
[the remark this answers is not in the document]
Comment (BCerki) [reply bc_obps/registration/api/operation.py:237]: Yup, thank you

File: bc_obps/registration/constants.py [hunk 4]
```diff
@@ -14,3 +14,5 @@
 BC_CORPORATE_REGISTRY_REGEX_MESSAGE = "BC Corporate Registry Number should be 1-3 letters followed by 7 digits."
 
 BORO_ID_REGEX = r"^\d{2}-\d{4}$"
+
+ALL_INDUSTRY_USERS = []
```
[the remark this answers is not in the document]
Comment (BCerki) [reply bc_obps/registration/constants.py:18]: Nowhere! I think I had an idea and abandoned it halfway

File: client/app/components/routes/select-operator/form/UserOperator.tsx (hunk 1 above)
Comment (BCerki) [reply client/app/components/routes/select-operator/form/UserOperator.tsx:119]: That makes sense, yes. I ~thought that was what I was doing here within the existing conditionality--is it not looking right for you?

Comment (Sepehr-Sobhani) [reply client/app/components/routes/select-operator/form/UserOperator.tsx:117]: It seems that the Note component is being generated even for new operators, which may not be the intended behavior. Please correct me if I'm wrong.

**Pull request merged**